std/net/curl.d

```diff
@@ -2836,6 +2836,40 @@ struct HTTP
         p.curl.set(CurlOption.useragent, userAgent);
     }
 
+    /**
+     * Get various timings defined in etc.c.curl.CurlInfo.
```

Comment:
Everything that's not English prose should be formatted specially, in this case as code - backticks for single words, `$(D ...)` for multiple words.